lib/render.js

```diff
@@ -0,0 +1,38 @@
+const _isEmpty = require('lodash/isEmpty'),
```

Comment:
would be nice to add `'use strict';` at the top, but not that big a deal

Reply:
I updated eslint and fixed this.